themes/compose/layouts/partials/scripts.html

```diff
@@ -19,3 +19,16 @@
         <script src="{{ $val }}"></script>
     {{ end }}
 {{ end }}
+
+<script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+<script>
+  if (window.netlifyIdentity) {
+    window.netlifyIdentity.on("init", user => {
+      if (!user) {
+        window.netlifyIdentity.on("login", () => {
+          document.location.href = "/admin/";
+        });
+      }
+    });
+  }
+</script>
```

Comment:
Out of curiosity, what would trigger this script when the user isn't on `/admin`?

Reply:
Check out this part of the doc https://www.netlifycms.org/docs/add-to-your-site/#authentication


> When a user logs in with the Netlify Identity widget, an access token directs to the site homepage. In order to complete the login and get back to the CMS, redirect the user back to the /admin/ path. To do this, add the following script before the closing body tag of your site's main index page:

Reply:
I think maybe i should not put these things in the scripts.html... since this is shared between all pages.

Reply:
https://github.com/taiwangoldcard/website/pull/122/commits/45154868a76c80e07d0b15fd76c60eda8dd9d6c4 here